command/node_status.go

```diff
@@ -333,6 +343,7 @@ func (c *NodeStatusCommand) formatNode(client *api.Client, node *api.Node) int {
 	}
 
 	if c.short {
+		basic = append(basic, fmt.Sprintf("Host Volumes|%s", strings.Join(nodeVolumes(node), ",")))
```

Comment:
Do we want to emit a line here if there is no node host volumes?

Reply:
This is maintaining the same behavior as for drivers right now - Omitting it entirely is probably more confusing than an empty list imo?

Reply:
I don't know what users expect.  Drivers field is almost never empty (otherwise, what's the point of clients), while host volumes is a feature that will take awhile to gain adoption

Reply:
Not showing them at all would be confusing imo, bc it could fairly easily lead to the confusion of e.g "oh am i running the wrong cli version" etc? - And in the format of output we use, I'm not sure there's really an "empty" entry that isn't an empty list that would make sense?

Reply:
makes sense - either way, nbd.